packages/bpk-docs/src/layouts/DocsLayout/DocsLayout.js

```diff
@@ -76,6 +76,7 @@ const links = [
       { id: 'NATIVE_INPUT', route: routes.NATIVE_INPUT, children: 'Text input' },
       { id: 'NATIVE_TEXT', route: routes.NATIVE_TEXT, children: 'Text' },
       { id: 'NATIVE_BUTTON', route: routes.NATIVE_BUTTON, children: 'Button' },
+      { id: 'NATIVE_CARD', route: routes.NATIVE_CARD, children: 'Cards' },
```

Comment:
being picky again: I would also update this to `NATIVE_CARDS`

Reply:
Actually, we should probably just deal with all the docs pluralisation inconsistencies separately.

Reply:
Well my mistake I'll fix it 😄